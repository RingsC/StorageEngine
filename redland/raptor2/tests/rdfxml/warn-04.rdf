<?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
         xmlns:ex="http://example.org/">

   <!-- check unknown rdf node element is picked up -->

  <rdf:description rdf:about="http://example.org/res">
    <ex:foo>bar</ex:foo>
  </rdf:description>
</rdf:RDF>
