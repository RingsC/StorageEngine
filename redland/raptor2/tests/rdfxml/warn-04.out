<http://example.org/res> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#description> .
<http://example.org/res> <http://example.org/foo> "bar" .
